js/modules/k6/html/elements_gen.go

```diff
@@ -1,4 +1,4 @@
-// generated by js/modules/k6/html/gen/gen_elements.go directed by js/modules/k6/html/elements.go;  DO NOT EDIT
+// generated by js/internal/modules/k6/html/gen/gen_elements.go directed by js/internal/modules/k6/html/elements.go;  DO NOT EDIT
```

Comment:
I think this is now wrong

Reply:
🤦  Thanks, I missed this. It's fixed in the latest push.